RELEASE-NOTES.md

```diff
@@ -67,6 +67,11 @@ Stanchion can be updated at any time during the system upgrade
 other instructions. Be careful enough not to run multiple live
 Stanchion nodes at once where both are referred from CS nodes.
 
+Any patches for CS 1.4 or 1.5 release cannot be applied to Riak CS
+2.0 because they need recompile as Riak CS 2.0 is shipped with different
+ERTS. Every official patches are supposed to be included in this
```

Comment:
Just for confirmation, "remove all patches" is for both riak and riak cs, right?
The list of steps above reads so, but this sentence almost only refer to Riak CS.